db/migrate/20170410130134_copy_server_id_from_jobs_to_miq_tasks.rb

```diff
@@ -0,0 +1,24 @@
+class CopyServerIdFromJobsToMiqTasks < ActiveRecord::Migration[5.0]
+  class Job < ActiveRecord::Base
+    belongs_to :miq_task, :class_name => 'CopyServerIdFromJobsToMiqTasks::Job'
+    self.inheritance_column = :_type_disabled
+  end
+
+  class MiqTask < ActiveRecord::Base
+    has_one :job, :class_name => 'CopyServerIdFromJobsToMiqTasks::MiqTask'
```

Comment:
There's not need to link these together because you don't actually use the relationship below.  Same goes for Job#miq_task belongs_to up above.